addon/components/base/bs-dropdown.js

```diff
@@ -293,6 +293,7 @@ export default class Dropdown extends Component {
   openDropdown() {
     this.set('isOpen', true);
     this.get('onShow')();
+    // schedule('afterRender', () => this.menuElement.focus());
```

Comment:
I guess just a left over from developing this one?

```suggestion
```

Reply:
oh, yes :)